### DISCUSSION TITLE
Problems with сompilation

### DISCUSSION THREAD

**Discussion by Korenovskiy** [ra3xdh/qucs_s#690] [Q&A]
When i write git submodule init, it says "Git repository not found", what should i do? thank you in advance

**Comment by ra3xdh** [comment]
I guess you have not cloned a source from git, but unpacked tarball. Tarball contains all submodules already inside.

**Comment by Korenovskiy** [comment]
i did't unpacked tarball, i have just wrote: "gir clone " and a link from github. Now it says: "submodule 'qucsator_rf' registered at path qucsator_rf", what should i do with that now?

**Comment by ra3xdh** [comment]
You should follow the installation instructions in the README.md https://github.com/ra3xdh/qucs_s?tab=readme-ov-file#compiling Enter the rest of commands to compile the application.

**Comment by Korenovskiy** [comment]
if i'm not mistaken after writing "make install" qucs-s is installed and we can open it. Am i right?

**Reply by wawuwo** [reply]
After `make install` the app binary should be in `your_install_prefix/bin/`. `your_install_prefix ` is the location you should have set when you were configuring the build (`cmake ..  -DCMAKE_INSTALL_PREFIX=/your_install_prefix/` in build instructions)

**Comment by ra3xdh** [comment]
No, `make install` only installs the program. You need to navigate to the directory where the application was installed and launch the executable.
